parisienne/README.md

```diff
@@ -0,0 +1,38 @@
+# Parisienne
+
+![A screenshot of the Parisienne theme.](https://user-images.githubusercontent.com/5375500/71043725-88134380-20fd-11ea-996f-44b6a1640856.png "Parisienne Screenshot")
+
+## About
+
+This theme an experiment using block-based templates and theme starter content to bootstrap a WordPress site. 
+
+[Demo](https://injured-anaconda.jurassic.ninja/)
```

Comment:
Ahh, we may want to change this to a more permanent demo sooner than later. Just an FYI.